.github/CONTRIBUTING.md

```diff
@@ -6,9 +6,8 @@ under the terms of [COPYING](../COPYING), which is an MIT-like license.
 ## Opening issues
 
 * Make sure you have a [GitHub account](https://github.com/signup/free)
-* [Submit an issue](https://github.com/NixOS/nixpkgs/issues) - assuming one does not already exist.
-  * Clearly describe the issue including steps to reproduce when it is a bug.
-  * Include information what version of nixpkgs and Nix are you using (nixos-version or git revision).
+* Search for existing issues on the topic
```

Comment:
wording isn't my strongest quality, but:
```suggestion
* Ensure an existing issue doesn't exist
```

" Search for" doesn't really imply an action (e.g. "Search for existing issues on the topic, and open a new one anyway")

Reply:
Yes, that should be more explicit. I'll improve it tomorrow.

Reply:
I improved it. Please check again!